samples/javascript_nodejs/25.logger-bot/README.md

```diff
@@ -42,18 +51,13 @@ In this example, the bot's state is used to track number of messages.
 # Deploy this bot to Azure
 You can use the [MSBot][5] Bot Builder CLI tool to clone and configure the services this sample depends on.
 
-To install all Bot Builder tools -
-
-Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher
-
-```bash
-npm i -g msbot chatdown ludown qnamaker luis-apis botdispatch luisgen
-```
+To install all Bot Builder tools, follow these [instructions][11].
```

Comment:
For consistency, can you change the instructions as we have it in [other samples](https://github.com/southworkscom/BotBuilder-Samples/commit/99f4189f5330c9d61dc76a2effaa3cc41d97d0f7#diff-671f2784abcf5c38d62133d9f710dbaaR41)?